app/models/procurement_area_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class ProcurementAreaPresenter
+  def initialize(procurement_area, api_client)
+    @procurement_area = procurement_area
+    @api_client = api_client
+  end
+
+  def id
+    procurement_area.id
+  end
+
+  def name
+    procurement_area.name
+  end
+
+  def memberships
+    member_organisations_for_procurement_area
+  end
+
+  private
+
+  attr_reader :procurement_area, :api_client
+
+  def member_organisations_for_procurement_area
+    organisations.select { |org| membership_uids.include? org.uid }
```

Comment:
You mean extending the API to do the filtering? That is a decision somewhat beyond the scope of this PR, I think.